src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -225,21 +231,25 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
                             {!!(!this.store.cancerTypes.isPending && !this.store.cancerStudies.isPending) && (
                                 <Observer>
                                     {() => {
+                                    const studyLimitReached = this.store.selectableSelectedStudyIds.length > 50;
```

Comment:
@khzhu i realized that this button has been eliminated (replaced by "forked mode" button).  You should rebase and then reapply this logic to the forked button.  Let me know if you need help. SHould be pretty clear when you rebase (or if you look at latest public portal)